specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10/examples/ReplicationRecoveryPlans_PlannedFailover.json

```diff
@@ -10,6 +10,7 @@
         "failoverDirection": "PrimaryToRecovery",
         "providerSpecificDetails": [
           {
+            "vaultLocation": "central-us",
```

Comment:
This is marked as obsolete in RecoveryPlanHyperVReplicaAzureFailoverInput.
Remove this.
 [Obsolete("This property is no longer used in failover flows.")]
 public string VaultLocation { get; set; }